pkg/builder/cbuildidx/builder_test.go

```diff
@@ -254,6 +254,12 @@ func TestBuildAllContexts(t *testing.T) {
 		assert.Nil(err)
 	})
 
+	t.Run("test build use context set", func(t *testing.T) {
+		b.Options.UseContextSet = true
```

Comment:
can we set this to false after we are done with the test.?
Because this will be propagated to the next tests as well.